src/components/pages/devices/flyouts/SIMManagement/SIMManagement.js

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React, { Component } from 'react';
+import { Trans } from 'react-i18next';
+import { Link } from 'react-router-dom'
+import { Select } from 'components/shared';
+
+
+import {
+  Flyout,
+  FlyoutHeader,
+  FlyoutTitle,
+  FlyoutCloseBtn,
+  FlyoutContent,
+} from 'components/shared';
+
+import './SIMManagement.css';
+
+const simManagementUrl = 'https://iot.telefonica.com/contact';
+
+const optionValues = [
+  { value: 'telefonica'}
+];
+
+const initialState = {
+  provider: 'telefonica'
+};
+
+export class SIMManagement extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = initialState;
+  }
+
+  onChange = (propOnChange) => ({ target: { value: { value } = {} } = {} }) => {
```

Comment:
The `propOnChange` parameters doesn't appear to be used so it doesn't seem like this function needs to be curried. 

That said, instead of defining your own onChange event it would be better to use [state links](https://github.com/Azure/pcs-remote-monitoring-webui/tree/master/src/utilities#linked-components) like in other parts of the app. This will handle the onChange and value bindings for you as well as providing easy validation logic (if it is needed).

In your imports, add
`import { LinkedComponent } from 'utilities';`

then change the class definition to 
`SIMManagement extends LinkedComponent`

inside your constructor you can then define a link
```
this.providerLink = this.linkTo('provider') // where "provider" is the name of the value in the state
  .map(({ value }) => value); // converts the Select value in state from { label, value } to just the value itself
```

then in the render method your select component becomes
```
// Note that we are using FormControl instead of Select directly
<FormControl
  type="select"
  className="sim-management-dropdown"
  options={options}
  searchable={false}
  clearable={false}
  link={this.providerLink} />
```